internal/telemetry/metrics.go

```diff
@@ -22,9 +22,11 @@ import (
 	"go.opencensus.io/tag"
 )
 
+// Default histogram distributions
 var (
-	// HistogramBounds defines a unified bucket boundaries for all histogram typed time metrics in Open Match
-	HistogramBounds = []float64{0, 50, 100, 200, 400, 800, 1600, 3200, 6400, 12800, 25600, 51200}
+	DefaultBytesDistribution        = view.Distribution(64, 128, 256, 512, 1024, 2048, 4096, 16384, 65536, 262144, 1048576)
+	DefaultMillisecondsDistribution = view.Distribution(0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000)
+	DefaultCountDistribution        = view.Distribution(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536)
```

Comment:
Should this have a value for 0?

Reply:
I'm following the usage at here https://github.com/census-instrumentation/opencensus-go/blob/master/plugin/ocgrpc/stats_common.go#L57